database/buildSchema/44_data_view.sql

```diff
@@ -12,8 +12,7 @@ CREATE TABLE data_view (
     detail_view_exclude_columns varchar[],
     detail_view_header_column varchar NOT NULL,
     show_linked_applications boolean NOT NULL DEFAULT TRUE,
-    priority integer DEFAULT 1,
-    UNIQUE (table_name, code)
```

Comment:
Remove constraint -- allows more than one view with the same code. This is now how we can have different views based on permission, but without duplicating entries in the list of available for views for users that happen to have multiple permissions